src/modules/types/science/biology/well_plate/view.js

```diff
@@ -166,7 +166,7 @@ define(['modules/default/defaultview', 'src/util/api', 'src/util/color'], functi
         if (!colorBySample && colorByJpath) {
           let arrayPath = colorJpath ? colorJpath.split('.') : undefined;
           arrayPath = arrayPath[arrayPath.length - 1];
```

Comment:
This should be in the previous line. You don't what to access an element in case it is undefined